sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/implementation/ClientFactory.java

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.config.implementation;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.azure.spring.cloud.config.AppConfigurationCredentialProvider;
+import com.azure.spring.cloud.config.ConfigurationClientBuilderSetup;
+import com.azure.spring.cloud.config.health.AppConfigurationStoreHealth;
+import com.azure.spring.cloud.config.properties.AppConfigurationProperties;
+import com.azure.spring.cloud.config.properties.AppConfigurationProviderProperties;
+import com.azure.spring.cloud.config.properties.ConfigStore;
+
+/**
+ * Manages all client connections for all configuration stores.
+ */
+public class ClientFactory {
```

Comment:
So the `ClientFactory` holds n `ConfigStore`, each `ConfigStore` hold 1 or n `ConfigurationClientWrapper`, and each `ConfigurationClientWrapper` maps to one `ConfigurationClient`?

Reply:
Yes